core/app/helpers/mno_enterprise/image_helper.rb

```diff
@@ -0,0 +1,22 @@
+module MnoEnterprise
+  module ImageHelper
+    # Helper method to easily access and select the images
+    def select_image
```

Comment:
The method name is pretty obscure.

I think the helper could be called something like `ImageHelper#main_logo_white_bg`?
That'd be a lot clearer.